File: src/test/java/ApplicationsNavigationTest.java
import org.testng.annotations.Test;

import static org.testng.Assert.assertEquals;
import static org.testng.Assert.assertTrue;
public class ApplicationsNavigationTest extends LoginTest {


    @Test
    public void applicationNavigationTest() {
 //       LoginPage loginPage = new LoginPage(driver);
//        loginPage.loginSuccessful("ribtestuser@gmail.com", "TestPassword#1");
        MainPage mainPage = new MainPage(driver);
        mainPage.navigateToApplications().applications().click();
        ApplicationsPage applicationsPage = new ApplicationsPage(driver);
        assertTrue(applicationsPage.tenderLink().isDisplayed());
    }
}

File: src/test/java/main/ApplicationsNavigationTest.kt
package main

import org.testng.Assert
import org.testng.annotations.Test

open class ApplicationsNavigationTest : LoginTest() {
    @Test
    fun applicationNavigationTest() {
        val mainPage = MainPage(driver)
        mainPage.navigateToApplications().applications().click()
        val applicationsPage = ApplicationsPage(driver)
        Assert.assertTrue(applicationsPage.tenderLink().isDisplayed)
    }
}



